llvm/tools/llvm-exegesis/lib/RISCV/RISCVCounters.cpp

```diff
@@ -0,0 +1,66 @@
+//===-- RISCVCounters.cpp ---------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
```

Comment:
Should you add a description of this file here like there are for other files ([example](https://github.com/llvm/llvm-project/blob/2834e8ad1cfe32b69a8b5a1dd2477725683a61c4/llvm/lib/Target/RISCV/RISCVISelLowering.cpp#L9))?

Reply:
Addressed